vm/src/format.rs

```diff
@@ -477,7 +477,17 @@ impl FormatSpec {
             },
             Some(FormatType::Number) => self.format_int_radix(magnitude, 10),
             Some(FormatType::String) => Err("Unknown format code 's' for object of type 'int'"),
-            Some(FormatType::Character) => Err("Unknown format code 'c' for object of type 'int'"),
+            Some(FormatType::Character) => match (self.sign, self.alternate_form) {
+                (Some(_), _) => Err("Sign not allowed with integer format specifier 'c'"),
+                (_, true) => {
+                    Err("Alternate form (#) not allowed with integer format specifier 'c'")
+                }
+                (_, _) => match num.to_u32() {
+                    Some(n) if n <= 0x10ffff => Ok(std::char::from_u32(n).unwrap().to_string()),
+                    // TODO: raise OverflowError
```

Comment:
Open an issue for this (and if you're working on it, tell me and I'll assign you). Leaving it in code as a TODO will probably mean it gets forgotten for a while.

Reply:
I've opened the issue. #4387
I'll tell you when I work on it.